Neos.Neos/Classes/Fusion/Cache/GraphProjectorCatchUpHookForCacheFlushing.php

```diff
@@ -110,28 +110,22 @@
  * => INVARIANT_2 violated.
  * => this case needs a cache flush at onBeforeBatchCompleted.
  *
- * SUMMARY: we need to flush the cache at BOTH places.
+ * SUMMARY: We need to flush the cache at BOTH places to ensure non of the cases above happens. BUT (!)
+ *          Flushing the cache is a very expensive operation, so we need to reduce them as much as we can.
+ *          So we decided to take the risk of CASE B, which might show an outdated content a bit longer than
+ *          it might be expected.
```

Comment:
Just wondering: Who is "we" in this case?

Reply:
The responsible "Cache Flushing Team of the Day". We rotate and everyone has to do their duty.

Reply:
😝

Reply:
tehe.. I didn't ask to disrespect anyone. I'm just missing some context as to _why_ this was decided this way (and, frankly, if I was part of it 😀 )

Reply:
Ahhh... you referring to the second "we" (decided to take the risk) ... 🙈 
I thought you and me discussed that, while we looked into the overhauling of the content cache flusher. But maybe I'm wrong and it was @kitsunet ... 🤔

Reply:
Yep that was me I think. :)